email-generator/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM node:alpine3.13
+
+WORKDIR /build
+COPY app/ .
+
+RUN npm install
```

Comment:
NPM version should be same as one we are recomming on desktop install - the latest nodejs LTS v14.x?